<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotWindowClass</class>
 <widget class="QWidget" name="BotWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>898</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>L2Bot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="widgetPlayerStatus" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelPlayerName">
         <property name="text">
          <string>PlayerName</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelTargetName">
         <property name="text">
          <string>TargetName</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelBotState">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QProgressBar" name="barHP">
         <property name="styleSheet">
          <string notr="true">QProgressBar::chunk { background-color: #FF5050; }</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="barTargetHP">
         <property name="styleSheet">
          <string notr="true">QProgressBar::chunk { background-color: #FF5050; }</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="buttonBotting">
         <property name="text">
          <string>Start Botting</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QProgressBar" name="barMP">
         <property name="styleSheet">
          <string notr="true">QProgressBar::chunk { background-color: #3366FF; }</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QProgressBar" name="barTargetMP">
         <property name="styleSheet">
          <string notr="true">QProgressBar::chunk { background-color: #3366FF; }</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="buttonClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetHolderMap" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="MapWidget" name="map">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>250</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkboxMapViewzone">
            <property name="text">
             <string>View targeting zone</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderZoom">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="pageStep">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="widgetTabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tabLog">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEditLog">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEntities">
       <attribute name="title">
        <string>Entities</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableNPC">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Entity ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>NPC ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Z</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabActions">
       <attribute name="title">
        <string>Combat Actions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableCombatActions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>EngageOnly</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Reuse delay</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboCombatActionType">
             <item>
              <property name="text">
               <string>Action Type</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Skill</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Action</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboCombatAction">
             <item>
              <property name="text">
               <string>Select an action...</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkboxCombatActionEngage">
             <property name="text">
              <string>Use only to engage target</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinboxCombatActionReusedelay">
             <property name="minimum">
              <number>300</number>
             </property>
             <property name="maximum">
              <number>9999000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonCombatActionAdd">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPartyActions">
       <attribute name="title">
        <string>Party actions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="tablePartyActions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Condition</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Reuse delay</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="1">
            <widget class="QComboBox" name="comboPartyAction">
             <item>
              <property name="text">
               <string>Action</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="comboPartyActionType">
             <item>
              <property name="text">
               <string>Action Type</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Skill</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Action</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="spinboxPartyActionReusedelay">
             <property name="minimum">
              <number>300</number>
             </property>
             <property name="maximum">
              <number>9999000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="comboPartyActionCondition">
             <property name="text" stdset="0">
              <string>Condition</string>
             </property>
             <item>
              <property name="text">
               <string>Condition</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="buttonPartyActionAdd">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditPartyActionCondition">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTargeting">
       <attribute name="title">
        <string>Targeting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupboxTargeting">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Target style</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="4" column="0">
            <widget class="QRadioButton" name="radioTargetingAssist">
             <property name="text">
              <string>Assist player</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioTargetingCircle">
             <property name="text">
              <string>Use circle</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioTargetingAnything">
             <property name="text">
              <string>Target anything</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QWidget" name="widget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Radius:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonTargetingSetcenter">
                <property name="text">
                 <string>Set zone center here</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinboxTargetingRadius">
                <property name="minimum">
                 <number>100</number>
                </property>
                <property name="maximum">
                 <number>30000</number>
                </property>
                <property name="value">
                 <number>4000</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditTargetingAssistName"/>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkboxTargetingSpoilSweep">
             <property name="text">
              <string>Spoil and sweep</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTableView" name="tableTargetingBlacklist">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Targeting zone center:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelTargetingCenter">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRest">
       <attribute name="title">
        <string>Rest</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Until MP &gt;=</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinboxRestHPMin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinboxRestMPMin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Until HP &gt;=</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkboxRestMP">
             <property name="text">
              <string>MP &lt;= (%)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="spinboxRestMPMax">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="spinboxRestHPMax">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkboxRestHP">
             <property name="text">
              <string>HP &lt;= (%)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>391</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QFrame</extends>
   <header>mapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="L2Bot.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>signal_buttonHookClicked(bool)</signal>
  <slot>slot_buttonHookClicked(bool)</slot>
 </slots>
</ui>
